dask/dataframe/io/json.py

```diff
@@ -157,9 +158,11 @@ def read_json(
         Text conversion, ``see bytes.decode()``
     compression : string or None
         String like 'gzip' or 'xz'.
-    engine : function object, default ``pd.read_json``
+    engine : callable or str, default ``pd.read_json``
         The underlying function that dask will use to read JSON files. By
         default, this will be the pandas JSON reader (``pd.read_json``).
+        If a string is specified, this value will be passed under the ``engine``
+        key-word argument to ``pd.read_json`` (only supported for Pandas-2.0).
```

Comment:
only supported starting from pandas-2.0?

Reply:
>only supported starting from pandas-2.0?

Does `pd.read_json` accept an engine argument for `pandas<2.0`? If so, my mistake.

Reply:
No, sorry. I meant rewording it. This sounds to me as it would only be accepted for 2.0 and not 2.1 for example

Reply:
>This sounds to me as it would only be accepted for 2.0 and not 2.1 for example

Ahah. good point!